Providers/Pocket/Model/PocketSponsoredStory.swift

```diff
@@ -0,0 +1,61 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+struct PocketSponsoredStory: Codable {
+    let id: Int
+    let flightId: Int
+    let campaignId: Int
+    let title: String
+    let url: String?
+    let domain: String
+    let excerpt: String
+    let priority: Int
+    let context: String
+    let rawImageSrc: URL
+    let imageURL: URL
+    let shim: Shim
+    let caps: Caps
+    let sponsor: String
+
+    enum CodingKeys: String, CodingKey {
+        case id = "id"
+        case flightId = "flight_id"
+        case campaignId = "campaign_id"
+        case title = "title"
+        case url = "url"
+        case domain = "domain"
+        case excerpt = "excerpt"
+        case priority = "priority"
+        case context = "context"
+        case rawImageSrc = "raw_image_src"
+        case imageURL = "image_src"
+        case shim = "shim"
+        case caps = "caps"
+        case sponsor = "sponsor"
+    }
+}
+
+struct PocketSponsoredRequest: Codable {
+    let spocs: [PocketSponsoredStory]
+}
+
+struct Shim: Codable {
+    let click: String
+    let impression: String
+    let delete: String
+    let save: String
+}
+
+struct Caps: Codable {
```

Comment:
If Shim and Caps only relate to Pocked Sponsored Story, we could enclose it in that struct just so it doesn't appear in autocompletion when writing code.

Reply:
Yes. It's mostly related to `PockedSponsoredStory`, but both `PockedSponsoredStory` and `PocketFeedStory` are mapped to `PocketStory` which also uses Shims. Of course I can use the as `PockedSponsoredStory.Shim`, but I'm thinking I'm coupling `PocketStory` with too much info. Wdyt?

Reply:
Yes. `Caps` are some limits on how many times we show the sponsored story. And `Shims` I guess we have to send them to the Sponsor Provider to confirm we showed the story. Something like that.

Reply:
I ended up adding them under `PocketSponsoredStory`.